### PR TITLE
cpcfs: init at 0.85.4

### PR DESCRIPTION
**Pull request by bignaux**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by bignaux** [comment]
@GrahamcOfBorg build cpcfs

File: pkgs/tools/filesystems/cpcfs/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  nativeBuildInputs = [ makeWrapper ncurses readline ronn ];
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace '-ltermcap' '-lncurses' \
+      --replace '-L /usr/lib/termcap' ' '
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
```
Comment (c0bw3b) [inline pkgs/tools/filesystems/cpcfs/default.nix]:
```suggestion
    runHook preInstall
    mkdir -p $out/bin
```

File: pkgs/tools/filesystems/cpcfs/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    mkdir -p $out/man/man1
+    cp cpcfs $out/bin
+    ronn --roff ../template.doc --pipe > $out/man/man1/cpcfs.1
```
Comment (c0bw3b) [inline pkgs/tools/filesystems/cpcfs/default.nix]:
```suggestion
    ronn --roff ../template.doc --pipe > $out/man/man1/cpcfs.1
    runHook postInstall
```

File: pkgs/tools/filesystems/cpcfs/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Manipulating CPC dsk images and files";
+    homepage = "https://github.com/derikz/cpcfs/" ;
+    license = "BSD-2-Clause";
```
Comment (c0bw3b) [inline pkgs/tools/filesystems/cpcfs/default.nix]:
```suggestion
    license = licenses.bsd2;
```

**Comment by bignaux** [comment]
Thanks for review.

**Review by c0bw3b — APPROVED** [review]
Builds and runs

**Pull request merged**